mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -376,38 +366,14 @@ PopulateParamsXDL::isValidBlockwiseGemmXDLOPS(const InitParamsXDL &param,
   if ((param.gemmNPerBlock % param.gemmNPerWave) != 0)
     return failure();
 
-  // TODO Remove. Note KPerBlock and KPack are independent tuning parameters.
-  // There's no need to check if they are divide exactly
-  if ((param.gemmKPerBlock % param.gemmKPack) != 0)
-    return failure();
```

Comment:
This was documented here: https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/475.

According to Jack's comment, it looks like the problematic assumption is getting rid of in our gridwise->blockwise refactoring. Have you bumped into any additional breakages because of this.

Feel free to re-assign that ticket to you and close it once this PR is merged

Reply:
I haven't seen breakages, I'll check the buggy config tomorrow before landing this.

Reply:
Yep, it works now